### PR TITLE
Article about EGH

### PR DESCRIPTION
**Pull request by Anavelyz**
#83 
@xmnlab @EverVino check the article, please.

Le falta la ilustración del header.

### CODE REVIEW

**Comment by xmnlab** [comment]
@fccoelho @eduardocorrearaujo do you have time to review this article?

File: pages/blog/epigraphhub/epigraphhub.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+- Pronósticos
+- Aprendizaje automático
+- Minería de textos
+- Análisis Geoespacial
+
+Esto se encuentra disponible para los lenguajes de programación [R](https://github.com/thegraphnetwork/r-epigraphhub/blob/main/epigraphhub.Rproj) y [Python](https://github.com/thegraphnetwork/epigraphhub_py).
+
+## ¿Cómo utilizar la plataforma EpiGraphHub?
+
+Este procedimiento es muy sencillo, debes acceder a su dirección web: https://epigraphhub.org/. Luego puedes crear un usuario o emplear la cuenta de invitado (usuario:guest y contraseña:guest).
```
Comment (xmnlab) [inline pages/blog/epigraphhub/epigraphhub.md]: Usar dash.epigraphhub.org en lugar de epigraphhub.org ... pronto este último va a ser usado apenas para la página web y no más para el superset

File: pages/blog/epigraphhub/epigraphhub.md [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+
+## ¿Cómo contribuir en EpiGraphHub?
+
+EpiGraphHub al ser un proyecto de código abierto crea espacios en los que es posible contribuir, su repositorio principal se encuentra en [GitHub](https://github.com/thegraphnetwork/EpiGraphHub#contributing) por lo que los issues y la forma en que podemos brindar nuestro granito de arena se encuentra expresado allí. Igualmente, si tienes dudas puedes acudir a Discord.
+
+En resumen, EpiGraphHub es una herramienta que nos permite acceder y analizar datos a través de sus servidores, lo cual logra que trabajemos sin preocuparnos por la cantidad y el espacio que pudiesen ocupar en nuestro computador. También podemos utilizar sus funcionalidades para analizar datos almacenados en local y obtener resultados interesantes que no percibimos a simple vista en los datos. Además de esto, los gráficos y dashboards que podemos obtener a través de la plataforma son muy llamativos y podemos interactuar fácilmente con la información, recordemos que provienen de Apache Echarts.
+
+Si ya le echaste un vistazo a este proyecto puedes dejarnos tus comentarios a cerca de lo que percibiste y si te gusto.
```
Comment (xmnlab) [inline pages/blog/epigraphhub/epigraphhub.md]:
```suggestion
Si ya le echaste un vistazo a este proyecto puedes dejarnos tus comentarios a cerca de lo que percibiste y si te gustó.
```

File: pages/blog/epigraphhub/epigraphhub.md (hunk 1 above)
Comment (xmnlab) [inline pages/blog/epigraphhub/epigraphhub.md]:
```suggestion
Este procedimiento es muy sencillo, debes acceder a su dirección web: https://dash.epigraphhub.org/. Luego puedes crear un usuario o emplear la cuenta de invitado (usuario:guest y contraseña:guest).
```

**Review by fccoelho — DISMISSED** [review]

**Comment by xmnlab** [comment]
Thanks for the review @fccoelho !
@EverVino do you want to do the last review?

**Comment by EverVino** [comment]
> Thanks for the review @fccoelho ! @EverVino do you want to do the last review?

It seems good to me. This will be  completed with the image files. @xmnlab 
@Anavelyz The recommended size of the header image is 530px height and a 1450px for max-width. You can check these resources to get a good image if you want  1) https://www.glazestock.com/ 2) https://drawkit.com/ 3) https://undraw.co/illustrations

**Comment by xmnlab** [comment]
thanks for working on that @Anavelyz !

**Pull request merged**